pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -621,6 +631,9 @@ func (cloud *CloudProvider) AttachDisk(ctx context.Context, project string, volK
 	}
 
 	op, err := cloud.service.Instances.AttachDisk(project, instanceZone, instanceName, attachedDiskV1).Context(ctx).Do()
+	if op != nil {
```

Comment:
let's move this block after the err != nil checks. same everywhere else.

Reply:
By moving the logging statement to after the err != nil checks, the if op != nil is no longer necessary, so I removed that.